kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java

```diff
@@ -9588,7 +9588,14 @@ public String getDropboxDisplayName(String siteId)
 		}
 
 		// return the current user's sort name
-		return userDirectoryService.getCurrentUser().getSortName();
+		String displayId = userDirectoryService.getCurrentUser().getDisplayId();
```

Comment:
Same chunk of code in 3 places, it should be refactored out into a method taking a user parameter. That makes it easier if someone needs to change it in the future.